test/functional/api/v2/smart_proxies_controller_test.rb

```diff
@@ -181,7 +181,7 @@ def setup_import_classes
   test "should obsolete puppetclasses" do
     setup_import_classes
     as_admin do
-      assert_difference('Environment.find_by_name("env1").puppetclasses.count', -2) do
+      assert_difference('Environment.find("env1").puppetclasses.count', -2) do
```

Comment:
Used to be this way prior to https://github.com/theforeman/foreman/commit/8387eb79d86947a63b3856c5ce64936f39c2a9a0